keps/77-dynamically-sized-jobs/kep.yaml

```diff
@@ -0,0 +1,24 @@
+title: 
+kep-number: 77
+authors:
+  - "@vicentefb"
+status: provisional
+creation-date: 2024-03-14
+reviewers:
+  - "@andrewsykim"
+  - "@alculquicondor"
+  - "@astefanutti"
+approvers:
+  - "@alculquicondor"
+  - "@andrewsykim"
+  - "@astefanutti"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v0.7"
+
+feature-gates:
+  - name: DynamicallySizedJobs
```

Comment:
Alternative names to consider:
* ElasticJobs
* ResizableJobs

Reply:
@andrewsykim I would go with ElasticJobs but since it's already referenced here https://github.com/kubernetes/enhancements/issues/3715 not sure if that could be an issue/confusion in future discussions